app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -79,21 +92,46 @@ class TableVisualization extends Visualization {
             return content
         }
 
+        function cellRenderer(params) {
+            if (params.value === null) {
+                return '<span style="color:grey; font-style: italic;">Nothing</span>'
+            } else if (params.value === undefined) {
+                return ''
+            } else if (params.value === '') {
+                return '<span style="color:grey; font-style: italic;">Empty</span>'
```

Comment:
what do you think about
```suggestion
                return '<span style="color:grey; font-style: italic;">Empty Text</span>'
```
or
```suggestion
                return '<span style="color:grey; font-style: italic;">""</span>'
```

`Empty` is not that clear to me that this is a text.

Personally I like `""` most.

But if you think that for data analysts `Empty` will be clearer, I'm not against it.